vnext/Microsoft.ReactNative/ReactNativeAppWindow.idl

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "DocString.h"
+
+namespace Microsoft.ReactNative
+{
+    [experimental]
+    DOC_STRING(
+        "A container for managing content of an application. It is wrapper for Microsoft.UI.Windowing.AppWindow."
+        "Additionally, it extends the AppWindow functionality to manage ReactNativeHost.")
+    runtimeclass ReactNativeAppWindow
```

Comment:
I didn't realize that Microsoft.UI.Windowing.AppWindow was sealed. is there some other default root application object that standard WinAppSDK apps use or do we really make all users create a AppWindow?